src/js/components/Video/__tests__/__snapshots__/Video-test.js.snap

```diff
@@ -163,8 +163,7 @@ exports[`Video autoPlay renders 1`] = `
 
 .c5:hover {
   background-color: rgba(221,221,221,0.4);
-  color: #444444;
-  color: #FFFFFF;
+  color: #000000;
```

Comment:
So I looked into this change, and the background color `rgba(221,221,221,0.4)` is translated to `#dddddd`, and the color-hex website using white color for its text for contrast instead of black as suggested in this line. 
I'm not a contrast expert but it might worth taking a look at this suggestion https://www.color-hex.com/color/dddddd

Reply:
regardless, this change is a potential backward compatibility candidate.

Reply:
I don't think this is a visible change, as the Video control Buttons don't have any visible text.

I will point out that https://contrast-ratio.com/#%23FFF-on-%23DDDDDD isn't accessible, but https://contrast-ratio.com/#%23000000-on-%23DDDDDD is.